fiasco/fiasco.py

```diff
@@ -86,7 +86,8 @@ def proton_electron_ratio(temperature: u.K, **kwargs):
             if ioneq is None:
                 warnings.warn(f'Not including {ion.ion_name}. Ionization fraction not available.')
                 continue
-            denominator += ioneq * abundance * ion.charge_state
+            denominator += ioneq * ion.charge_state
+        denominator *= abundance
```

Comment:
I don't think this is equivalent. This multiplies the abundance for that element on all existing terms. E.g. if you had summed over H and then did the sum over He, the He abundance would get multiplied by the H terms which is not correct.

Reply:
Ah yes, will revert this commit.